build/contracts/AutoInvestor.json

```diff
@@ -1,72 +0,0 @@
-      "type": "constructor"
-    },
-    {
-      "anonymous": false,
-      "inputs": [
-        {
-          "indexed": true,
-          "name": "loan",
-          "type": "address"
-        }
-      ],
-      "name": "Invested",
-      "type": "event"
-    }
-  ],
-  "unlinked_binary": "0x6060604052341561000f57600080fd5b6040516060806104498339810160405280805191906020018051919060200180519150505b60008054600160a060020a03808616600160a060020a0319928316179092556001805485841690831617905560028054928416929091169190911790555b5050505b6103c4806100856000396000f300606060405263ffffffff60e060020a60003504166303f9c7938114610024575b600080fd5b341561002f57600080fd5b610043600160a060020a0360043516610045565b005b6000805481908190600160a060020a0390811690851663b2016bd483604051602001526040518163ffffffff1660e060020a028152600401602060405180830381600087803b151561009657600080fd5b6102c65a03f115156100a757600080fd5b50505060405180519050600160a060020a03161415156100c657600080fd5b83600160a060020a031663c710ccd66000604051602001526040518163ffffffff1660e060020a028152600401602060405180830381600087803b151561010c57600080fd5b6102c65a03f1151561011d57600080fd5b5050506040518051600154909450600160a060020a038086169116149050806101535750600254600160a060020a038481169116145b151561015e57600080fd5b82915083600160a060020a031663ca5f42826000604051602001526040518163ffffffff1660e060020a028152600401602060405180830381600087803b15156101a757600080fd5b6102c65a03f115156101b857600080fd5b5050506040518051915050600160a060020a038216638067271a6000604051602001526040518163ffffffff1660e060020a028152600401602060405180830381600087803b151561020957600080fd5b6102c65a03f1151561021a57600080fd5b5050506040518051821115905061023057600080fd5b81600160a060020a03166324d427a93060405160e060020a63ffffffff8416028152600160a060020a039091166004820152602401600060405180830381600087803b151561027e57600080fd5b6102c65a03f1151561028f57600080fd5b50505081600160a060020a031663095ea7b3858360006040516020015260405160e060020a63ffffffff8516028152600160a060020a0390921660048301526024820152604401602060405180830381600087803b15156102ef57600080fd5b6102c65a03f1151561030057600080fd5b50505060405180515050600160a060020a03841663e8b5e51f6040518163ffffffff1660e060020a028152600401600060405180830381600087803b151561034757600080fd5b6102c65a03f1151561035857600080fd5b50505083600160a060020a03167fcbb217ba4b5dcd6d9c1ecf52e874be5dc26237e9f198d0f086097c419333366b60405160405180910390a25b505050505600a165627a7a72305820ee670168587561af0d2b64eff2f6f0e142442c0128b370feb39c03a5c8666ffa0029",
-  "networks": {
-    "42": {
-      "events": {
-        "0xcbb217ba4b5dcd6d9c1ecf52e874be5dc26237e9f198d0f086097c419333366b": {
-          "anonymous": false,
-          "inputs": [
-            {
-              "indexed": true,
-              "name": "loan",
-              "type": "address"
-            }
-          ],
-          "name": "Invested",
-          "type": "event"
-        }
-      },
-      "links": {},
-      "address": "0x33f96e1c74f73c125dcb643be52cd2728e32216d",
```

Comment:
Co ze zdeployowanymi kontraktami na sieciach testowych?
Te build'y są potrzebne aby się z nimi komunikować

Reply:
jeśli to jest tak jak teraz to będą mega conflicty jeśli będzie więcej niż jeden developer pracował nad tym samym contractem.